app/helpers/shop_helper.rb

```diff
@@ -27,9 +27,19 @@ def shop_tabs
       { name: 'producers', title: t(:shopping_tabs_producers), show: true },
       { name: 'contact', title: t(:shopping_tabs_contact), show: true },
       { name: 'groups', title: t(:shopping_tabs_groups), show: show_groups_tabs? },
+      custom_tab,
     ].select{ |tab| tab[:show] }
   end
 
+  def custom_tab
+    {
+      name: current_distributor.custom_tab&.title&.parameterize,
```

Comment:
What happens if I call my tab `shop`? I guess that at this point in time we can just call it `custom` because we have only one. In the future, we could maybe do a `custom_` prefix and then the title.

Reply:
Another way to handle it would be to validate the title and prevent any reserved names (home, shop, about, etc). Maikel's simple solutions would be fine too.

But in any case, it would be a misuse of the system to call the tab `shop`, so I think it's fine to leave it as-is.

Reply:
I wanted to use the tab title itself in order to be as _beautiful_ as possible, but I didn't think about the name collision. I'd like to use an _in between_ solution: adding the `custom_` prefix. 
e2b9a42044